examples/text_summarization/summarization_evaluation.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>This is really nice. I do remember some setup is required for using cpan.  Not sure if that can be automated. Or maybe at least mention it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/526/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Do you remember what else needs to be done to use cpan? I've tested these steps on two machines. </p>

Reply:
just config the cpan.

Reply:
Any specific commands? I don't remember anything else besides these commands. If you don't remember, we can add it later if people run into problems using these commands.